Hindawi Publishers.js

```diff
@@ -80,7 +89,27 @@ function doWeb(doc,url)
 				// use article ID as a page (seems to be the last part of URL/DOI)
 				item.pages = 'e' + item.DOI.substr(item.DOI.lastIndexOf('/') + 1);
 			}
-			item.extra = "";
+			//remove duplicate metadata			
+			for (i in item){
+				if (typeof item[i] == "string" && item[i].match(/^.+,/)){
+					//Z.debug(item[i])
+					if (item[i].match(/^(?:PMID: )?(.+),/)[1] == item[i].match(/, (.+)$/)[1]){
+						item[i] = item[i].replace(/,.+?$/, "")
+					}
+				}
+			}
+			//remove duplicate authors
+			if (item.creators.length%2 == 0){
+				var j=item.creators.length -1
+				for (var i = item.creators.length/2 -1; i>=0; i--, j--){
+					if (item.creators[i].firstName === item.creators[j].firstName &&
+						item.creators[i].lastName === item.creators[j].lastName){
+							item.creators.splice(j, 1);
+						}
+				}
+			}
```

Comment:
I'm guessing that the authors are either all duplicated or none of them are, right? It doesn't make much sense otherwise. If so, I think this should be cleaner.
```
if (item.creators.length && item.creator.length % 2 == 0) {
  var duplicate = true;
  for (var i = 0, j = item.creators.length / 2; duplicate && j < item.creators.length; i++, j++) {
    var a1 = item.creators[i], a2 = item.creators[j];
    duplicate = a1.firstName == a2.firstName && a1.lastName == a2.lastName;
  }

  if (duplicate) {
    item.creators = item.creators.slice(0, item.creators.length / 2);
  }
}
```